endtoendtests/src/main/java/com/microsoft/azure/functions/endtoend/HttpTriggerTests.java

```diff
@@ -47,11 +47,6 @@ public HttpResponseMessage HttpTriggerJava(
         }
 
         Gson a = new Gson();
```

Comment:
Why this instantiation is here?

Reply:
I just wanted to make sure that there is no class exception. I will add more comments.

Reply:
Add comment is fine.